lib/active_merchant.rb

```diff
@@ -59,10 +59,10 @@ def Base64.strict_encode64(v)
 require 'timeout'
 require 'socket'
 
-require 'active_utils/common/network_connection_retries'
-silence_warnings{require 'active_utils/common/connection'}
-require 'active_utils/common/post_data'
-require 'active_utils/common/posts_data'
+require 'active_merchant/network_connection_retries'
+silence_warnings { require 'active_merchant/connection' }
```

Comment:
I fixed the warnings - they were caused by Connection, and NetworkConnectionRetries both defining a `retry_safe` accessor.